### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by jaylinski**
Translation was provided by MB1968 (forum.kodi.tv).

### CODE REVIEW

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+msgstr "Nieuw & Populair"
+
+# GUI - Generic
+msgctxt "#30901"
+msgid "Next page"
+msgstr "Volgende pagina"
+
+msgctxt "#30902"
+msgid "Not available in your country"
+msgstr "Niet beschikbaar in Nederland"
```
Comment (jaylinski) [inline resources/language/resource.language.nl_nl/strings.po]: This assumes that the Dutch language is only used inside the Netherlands.

Comment (Markman-B) [reply resources/language/resource.language.nl_nl/strings.po]: And Belgium.

Comment (jaylinski) [reply resources/language/resource.language.nl_nl/strings.po]: @Markman-B Can you provide a better matching translation for the string `"Not available in your country"`? That would be great.

Comment (Markman-B) [reply resources/language/resource.language.nl_nl/strings.po]: I already tried so, but I have no permission on Github to edit the language file. But the correction is easy. Just replace "Nederland" by "jouw land" (without quotation marks).

Comment (jaylinski) [reply resources/language/resource.language.nl_nl/strings.po]: Thank you!

**Pull request merged**